php/php.blade/src/org/netbeans/modules/php/blade/editor/components/ComponentElement.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.php.blade.editor.components;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * TO use in implementation with ComponentModel
+ * 
+ * @author bhaidu
+ */
+public class ComponentElement {
+    public String name;
+    public String qualifiedClassName;
+    
+    public Set<String> properties = new HashSet<>();
```

Comment:
Please reconsider visibility.

Reply:
Indeed,
it was part of an unfinished evol, I will remove the **ComponentModel** and **ComponentElement**.